packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,139 @@
+import { ECIES, PhoneNumberUtils, SignatureUtils } from '@celo/utils'
 import BigNumber from 'bignumber.js'
+import { zip } from 'lodash'
```

Comment:
sneaky nam

Reply:
`import { zip } from '@celo/utils/lib/collections'` ;)

Reply:
:) I added a `zip3` to collections, the existing zip wasn't quite working for me, because:

1. It expected a function as a first argument. IMO, it should just let consumers do that themselves with a map?
2. It can only add two collections, not more than that.

```
export function zip3<A, B, C>(as: A[], bs: B[], cs: C[]) {
  const len = Math.min(as.length, bs.length, cs.length)
  const res: Array<[A, B, C]> = []

  for (let i = 0; i < len; i++) {
    res.push([as[i], bs[i], cs[i]])
  }
  return res
}

```